LINUX.md

```diff
@@ -0,0 +1,46 @@
+# Building on Linux
+Building on linux is made possible by cctools-port.
+
+Here is how, exactly, to install cctools and use it to compile.
+
+1. Dependencies
+- You need: clang (>=3.4) and everything Procursus normally needs. Get clang with apt.
+
+2. Get iOS SDK
+- You're going to need an iOS 13 SDK. Since you're on Linux, you're going to have to search online for one. I personally use https://github.com/xybp888/iOS-SDKs.
```

Comment:
It's best to use an unpatched SDK, you'll also want to copy the c++ stdlib into in, which you will get from the Xcode tool chain

Reply:
I have no idea how or where to get that. Please suggest an edit on exactly how.

Reply:
Take a look at the script I sent, it has all this taken care of.